### PR TITLE
Feature 4509 - Contents - Web Page

### PR DESCRIPTION
**Pull request by kboczkowski-clurgo**
## Description
I've prepared base component for geostory

## Issues
 - #4509 
 - ...

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [ ] Bugfix
 - [x] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [ ] Yes, and I documented them in migration notes
 - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: web/client/components/geostory/contents/WebPage.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```
Comment (MV88) [inline web/client/components/geostory/contents/WebPage.jsx]: missing copyrights

File: web/client/components/geostory/contents/__tests__/WebPage-test.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```
Comment (MV88) [inline web/client/components/geostory/contents/__tests__/WebPage-test.jsx]: missing copyrights

File: web/client/components/geostory/contents/WebPage.jsx (hunk 1 above)
Comment (kboczkowski-clurgo) [reply web/client/components/geostory/contents/WebPage.jsx]: done

File: web/client/components/geostory/contents/__tests__/WebPage-test.jsx (hunk 2 above)
Comment (kboczkowski-clurgo) [reply web/client/components/geostory/contents/__tests__/WebPage-test.jsx]: done

**Pull request merged**